indico/modules/events/surveys/util.py

```diff
@@ -38,3 +38,17 @@ def make_survey_form(questions):
             continue
         setattr(form_class, name, field_impl.get_wtforms_field())
     return form_class
+
+
+def create_preview_form(questions):
+    """Creates a WTForm from survey questions
+
+    This form is intended to be used as a preview form of a survey
+
+    :return An `IndicoForm` subclass
+    """
+    form_class = type(b'SurveySubmissionForm', (IndicoForm,), {})
```

Comment:
that class name, even though it's only used when e.g. looking at a repr, doesn't really fit for a *preview* form